wrappers/src/sync_session_cs.cpp

```diff
@@ -71,6 +71,7 @@ REALM_EXPORT CSharpSessionState realm_syncsession_get_state(const SharedSyncSess
         switch (session->state()) {
         case SyncSession::State::Inactive:
         case SyncSession::State::Dying:
+        case SyncSession::State::Paused:
```

Comment:
I suppose changes in this file come from the updates coming from core but have nothing to do with the PR itself. Correct?

Reply:
Correct - was just pulling latest Core to pick up @JacobOscarGunnarsson's changes and those flowed as side effects.